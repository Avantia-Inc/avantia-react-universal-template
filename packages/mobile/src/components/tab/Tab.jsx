import React from 'react';
import { Title } from 'react-native-paper';

const Tab = props => {
  return <Title>{props.title}</Title>;
};

export default Tab;
