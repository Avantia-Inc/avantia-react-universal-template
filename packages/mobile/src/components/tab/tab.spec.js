import React from 'react';
import { shallow } from 'enzyme';

// Component
import Tab from './Tab';

// Render the component.
it('should render the Tab component', () => {
  expect(shallow(<Tab />)).toMatchSnapshot();
});
