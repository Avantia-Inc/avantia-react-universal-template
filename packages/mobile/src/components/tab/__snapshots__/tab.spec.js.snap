// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Tab component 1`] = `<Component />`;
